src/telliot_feeds/pricing/price_service.py

```diff
@@ -34,7 +34,7 @@ async def get_price(self, asset: str, currency: str) -> OptionalDataPoint[float]
         raise NotImplementedError
 
 
-class WebPriceService(PriceServiceInterface, ABC):
+class WebPriceService(PriceServiceInterface):
```

Comment:
Is this to get rid of the `tox -e style` errors? Thank you sire, and just add some info about this to your PR summary next time